sklearn/preprocessing/tests/test_data.py

```diff
@@ -701,6 +703,62 @@ def test_scaler_without_centering():
     assert_array_almost_equal(X_csc_scaled_back.toarray(), X)
 
 
+def _check_attributes_scalers(scaler_1, scaler_2):
+    assert scaler_1.mean_ == scaler_2.mean_
```

Comment:
don't we need array equality?

Reply:
since that this is for the case where `mean_` will be None, I don't think so.

Reply:
I would rather make the check explicit in this case:

```python
assert scaler_1.mean_ is scaler_2.mean_ is None
```

(I did not know that such ternary identity assertions would be valid python but it seem to be the case :)